lib/bsdlib/bsd_os.c

```diff
@@ -437,9 +437,10 @@ void bsd_os_init(void)
 int32_t bsd_os_trace_put(const uint8_t * const data, uint32_t len)
 {
 #ifdef CONFIG_BSD_LIBRARY_TRACE_ENABLED
-	/* FIXME: Due to a bug in nrfx, max DMA transfers are 255 bytes. */
-
-	/* Split RAM buffer into smaller chunks to be transferred using DMA. */
+	/* Max DMA transfers are 255 bytes.
+	 * Split RAM buffer into smaller chunks
```

Comment:
Wouldn't this comment fit into a single line?

Reply:
Yes barely, but then it'd be much longer than the second line and it would look bad 😶